plugins/dfu/dfu-target.c

```diff
@@ -472,6 +474,52 @@ dfu_target_status_to_error_msg (DfuStatus status)
 	return NULL;
 }
 
+static gboolean
+dfu_target_poll_in_manifest (DfuTarget *target, GError **error)
```

Comment:
I think you want to call it something like `dfu_target_manifest_poll` or  `dfu_target_manifest_wait`

Reply:
Okay. I will change it to dfu_target_manifest_wait.